src/main/java/com/foundation/search/view/ParametersPanel.java

```diff
@@ -317,7 +360,128 @@ public JButton getCleanButton(){
         return cleanButton;
     }
 
-     /**
+    /**
+     * This method initialize all input labels, combo boxes and check boxes
+     * displayed in this panel
+     *
+     */
+    public void initializeInputs(){
+        //initializing the inputs, comboboxes and checkboxes
+        nameInput = new JTextField();
+        String[] extensions = {" ","exe", "png","mp4","pdf","mp4","jpg","txt","xls",
```

Comment:
There are 2 "mp4", replace one with a different extension name.
Also "xcls", I think it is not an extension, could you replace this with xlsx?

Reply:
Done, thanks